apps/fhir/server/authentication.py

```diff
@@ -1,12 +1,35 @@
 import requests
 from django.conf import settings
+from oauth2_provider.models import get_application_model
+from django.contrib.auth import get_user_model
 from rest_framework import exceptions
 from apps.dot_ext.loggers import get_session_auth_flow_trace
 from apps.fhir.bluebutton.utils import generate_info_headers
 from ..bluebutton.exceptions import UpstreamServerException
 from ..bluebutton.utils import (FhirServerAuth,
                                 get_resourcerouter)
 from .loggers import log_match_fhir_id
+from apps.fhir.bluebutton.signals import (
+    pre_fetch,
+    post_fetch
+)
+
+
+def get_app_and_org(client_id):
```

Comment:
@JFU-GIT Per removing the related code in apps/fhir/server/authentication.py, this function can be removed also.